src/intl.js

```diff
@@ -8,16 +8,20 @@ const intlStrings = {
     'inputSchema.validation.objectValuesInvalid': 'Keys [{invalidKeys}] in input.{fieldKey} should have string value which matches pattern "{pattern}"',
     'inputSchema.validation.additionalProperty': 'Property {fieldKey} is not allowed.',
     'inputSchema.validation.proxyGroupsNotAvailable': 'You currently do not have access to proxy groups: {groups}',
+    'inputSchema.validation.customProxyInvalid': 'Proxy URL {invalidUrl} is not valid.',
```

Comment:
We could udpate message `Proxy URL {invalidUrl} doesn't have valid format http://[URL]:[PORT]